metabase/metabase
# add line symbol on hover

### Description

With dc.js even when a line series did not have circles we showed the circle of the hovered datum which emphasizes what point is hovered. This PR returns this behavior.

### How to verify

- Create a line chart without circles on the line
- Hover data points
- Ensure it shows a circle for the hovered data point

### Demo

Before
<video src="https://github.com/metabase/metabase/assets/14301985/4f4ae130-4ffe-4159-8724-4e55b743fa8b" />

After
<video src="https://github.com/metabase/metabase/assets/14301985/37071935-f36c-40d0-b97d-79b8402679b1" />
